singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -344,6 +392,9 @@ inline SpinerEOSDependsRhoSie::SpinerEOSDependsRhoSie(const std::string &filenam
 
   status += loadDataboxes_(matid_str, file, lTGroup, lEGroup, coldGroup);
 
+  Transform_data_ = TransformData(lRhoOffset_, sieCold_);
```

Comment:
I _think_ we want to pass by reference here
```suggestion
  Transform_data_ = TransformData(lRhoOffset_, &sieCold_);
```

Reply:
That's not what this will do. You'll be passing in a pointer if you make this change. Passing by reference is declared in the function signature. The original code is fine.